test/acceptance/features/customEnvVar.feature

```diff
@@ -7,11 +7,11 @@ Feature: Inject custom env variable into application
     Background:
         Given Namespace [TEST_NAMESPACE] is used
         * Service Binding Operator is running
+        * OLM Operator "backend" is running
```

Comment:
If we have OLM operator as backend in the Background then like previous feature files we can technically have olm as a tag to the entire feature file itself.